### PR TITLE
add cannon,rsvp,wonder-commonlib,wonder-frp dts file

### PR DESCRIPTION
**Pull request by yyc-git**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: cannon/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (mhegazy) [inline cannon/tsconfig.json]: `true` instead. applies to other config files as well.

File: cannon/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,989 @@
+declare module CANNON {
+
```
Comment (mhegazy) [inline cannon/index.d.ts]: Please add a tslint.json, see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#lint

File: cannon/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,989 @@
+declare module CANNON {
```
Comment (mhegazy) [inline cannon/index.d.ts]: Missing header. also applies to other files.

just run `npm run new-package cannon`

seee https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#create-a-new-package

A header looks like:

```ts
// Type definitions for cannon <version>
// Project: https://github.com/beatgammit/base64-js
// Definitions by: YYC <https://github.com/yyc-git>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by yyc-git** [comment]
Hello! I have pass the ci, please check and merge the request! 
Thanks very much!

File: cannon/index.d.ts (hunk 2 above)
Comment (yyc-git) [inline cannon/index.d.ts]: please  pass!

**Comment by mhegazy** [comment]
Thanks for the updates, can you please also add the `tslint.json`, and fix the lint warning you will get. see: https://github.com/DefinitelyTyped/DefinitelyTyped#lint

**Comment by yyc-git** [comment]
Nice to see your reply!
The lint warn is difficult to fix, so can i not add the tslint.json?please?

**Comment by yyc-git** [comment]
Please merge the request so that I can get the dts from npm~

**Comment by horiuchi** [comment]
👍

**Comment by yyc-git** [comment]
Thanks very much!

**Pull request merged**